test/e2e/check_upgrades.go

```diff
@@ -143,7 +143,7 @@ func MajorUpgradeToV2(logger *log.Logger) error {
 	defer cancel()
 
 	logger.Println("Creating testnet")
-	testnet, err := testnets.New("runMajorUpgradeToV2", seed, nil)
+	testnet, err := testnets.New("runMajorUpgradeToV2", seed, nil, app.DefaultInitialVersion)
 	testnets.NoError("failed to create testnet", err)
 
 	defer testnet.Cleanup()
```

Comment:
I was thinking about that since i created that function when changing default consensus params but i thought that greater control over the initial version of app was useful because sometimes we want to start with v2.

Reply:
But we don't need to add the argument to the constructor.

If we want to change the app version we just call `testnet.SetConsensusParams`

Reply:
okay changing it to your suggestion. thanks